src/core/layout/qgslayoutitemelevationprofile.cpp

```diff
@@ -960,6 +967,9 @@ void QgsLayoutItemElevationProfile::recreateCachedImageInBackground()
   mPainter.reset( new QPainter( mCacheRenderingImage.get() ) );
 
   QList< QgsAbstractProfileSource * > sources;
+  QList< QgsAbstractProfileSource * > registrySources = QgsApplication::profileSourceRegistry()->profileSources();
+
+  sources << registrySources;
```

Comment:
```suggestion
  sources << QgsApplication::profileSourceRegistry()->profileSources();
```

Reply:
Addressed in https://github.com/qgis/QGIS/pull/57311/commits/de9abdfeaa806000a060d22b308048dd245beb75